src/components/ActivityFeed/ActivityFeedItem.js

```diff
@@ -241,11 +238,10 @@ export class ActivityFeedItem extends React.Component<Props> {
     } = this.props;
 
     if (isForAllAccounts) {
-      const isBetweenAccounts = (isSWAddress(to, accounts) && isKWAddress(from, accounts))
-        || (isSWAddress(from, accounts) && isKWAddress(to, accounts));
-      return (isBetweenAccounts && isReceived)
-        || (!isBetweenAccounts && isKWAddress(to, accounts))
-        || (!isBetweenAccounts && isSWAddress(to, accounts));
+      const isBetweenAccounts = isSmartWalletAccountAddress(to, accounts)
+        && isSmartWalletAccountAddress(from, accounts);
```

Comment:
The logic here differs significantly form previous check which was checking whether the transactions was KW <-> SW, and now it checks SW <-> SW. Is this the intent here?

Reply:
We do not support KW anymore and it shouldn't stay in accounts that's why this check leaves us only with Smart Wallets (Archanova<>Etherspot), hope it makes sense.